introduction.md

```diff
@@ -1,2 +1,5 @@
 Luis, 24 years old Informatics Engineer
-Favorite codeschool path is Ruby!
\ No newline at end of file
+Favorite codeschool path is Ruby!
+* Git
+* Java
+* Others
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?